src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Rules/CannotChangeVisibility.cs

```diff
@@ -37,8 +47,8 @@ public CannotChangeVisibility(RuleSettings settings, IRuleRegistrationContext co
                 return;
             }
 
-            Accessibility leftAccess = left.DeclaredAccessibility;
-            Accessibility rightAccess = right.DeclaredAccessibility;
+            Accessibility leftAccess = normalizeAccessibility(left.DeclaredAccessibility);
+            Accessibility rightAccess = normalizeAccessibility(right.DeclaredAccessibility);
 
             if (leftAccess > rightAccess)
```

Comment:
When someone is looking at internal members, this would not flag an API breaking change from `protected` to `internal`